addons/sys_intercom/fnc_enterVehicle.sqf

```diff
@@ -25,20 +25,18 @@ if (_unit != _vehicle) then {
     // Save vehicle
     _unit setVariable [QGVAR(intercomVehicle), _vehicle];
 
-    // Save current seat variable
```

Comment:
Here the seat information was stored for the unit. Why is this removed? I initially wanted to avoid calling it in a PFH and decided to check it and update it only once whenever it was necessary.

Reply:
> Here the seat information was stored for the unit. Why is this removed?

This was redundant. It got saved in intercom PFH anyways right after. All it did was run unnecessary procedure in `seatSwitched` which is for invalidation when leaving a certain seat (which can't happen when you enter a vehicle).

> I initially wanted to avoid calling it in a PFH and decided to check it and update it only once whenever it was necessary.

That is not how it worked.